docs/source/kedro_project_setup/settings.md

```diff
@@ -4,28 +4,36 @@ A Kedro project's `settings.py` file contains the application configuration for
 
 By default, all code in `settings.py` is commented out. In the case that settings are not supplied, Kedro chooses sensible default values. You only need to edit `settings.py` if you wish to change to values other than the defaults.
 
-## Available settings
+## Lifecycle settings
```

Comment:
Don't try to understand this diff... Just look at the picture in the PR description instead.